controller-api/src/main/java/com/yahoo/vespa/hosted/controller/api/role/RoleDefinition.java

```diff
@@ -20,60 +20,46 @@
     /** Deus ex machina. */
     hostedOperator(Policy.operator),
 
+    /** Reader - the base role for all tenant users */
+    publicReader(
+            Policy.tenantRead,
+            Policy.applicationRead,
+            Policy.deploymentRead,
+            Policy.publicRead
+    ),
+
+    /** User - the dev.ops. role for normal Vespa tenant users */
+    publicDeveloper(
+            Policy.applicationCreate,
+            Policy.applicationUpdate,
+            Policy.applicationDelete,
+            Policy.developmentDeployment
+    ),
+
+    /** Admin - the administrative function for user management etc. */
+    publicAdministrator(
+            Policy.tenantUpdate,
+            Policy.tenantManager
+    ),
+
+    /** Headless - the application specific role identified by deployment keys for production */
+    publicHeadless(
```

Comment:
I think the `public` prefix could be dropped from all these names. 

The `publicRead` privilege refers to the common word "public", not the system(s) :)

Reply:
As I mentioned before - I am not a huge fan of the names in the enum, so I am all for removing the prefix.

Reply:
Great!